Vagrantfile

```diff
@@ -31,6 +31,7 @@ Vagrant.configure(2) do |config|
   config.vm.define "#{project_name}" do |node|
       node.vm.hostname = "#{project_name}#{rand(01..99)}"
       node.vm.network :private_network, type: "dhcp"
+      #node.vm.network :private_network, :ip => '10.10.10.10', :auto_network => true
```

Comment:
I think it should probably be removed completely. I don't think we want that in there at all. I was just curious if there was a reason you had added it.